docroot/themes/humsci/su_humsci_theme/scss/base/_global.scss

```diff
@@ -232,3 +232,28 @@ body *:focus {
 .decanter-font-lead {
   max-width: 100%;
 }
+
+@-moz-keyframes spin {
```

Comment:
Your sass compiler could benefit from an autoprefixer so you wouldn't have to add them yourself.  
https://www.npmjs.com/package/autoprefixer

Reply:
oh thats already there, maybe i just need 
```spin {
  100% {
    -webkit-transform: rotate(360deg);
  }
}```
???

Reply:
If you want to animate you will still need that property but you will ony need one keyframe declaration

https://github.com/SU-SWS/decanter/pull/443/files#diff-66c59ec628d07d6fc01b96308e6094e5R28

Reply:
hmm. i see autoprefixer in the Gruntfile.js but it just compiles to 
```
@keyframes spin {
  100% {
    transform: rotate(360deg);
  }
}
```
maybe i'm doing something wrong

Reply:
That probably means it is compatible with the browsers we support

https://caniuse.com/#search=keyframe

Reply:
oh ha! look at that, it still works. :+1: